src/components/Frame/types.ts

```diff
@@ -63,4 +64,5 @@ export interface ToastProps {
   error?: boolean;
   /** Callback when the dismiss icon is clicked */
   onDismiss(): void;
+  action?: Action;
```

Comment:
The Polaris style guide renders the comment above a prop as the prop description. Something like...

```suggestion
/** An action to render beside the dismiss button */
  action?: Action;
```